src/ldapHelper.py

```diff
@@ -9,6 +9,8 @@ def __init__(self, ldapUri, ldapBindDn, ldapBindPassword, ldapBaseDn):
 
     def bind(self):
         try:
+            # uncomment to disable CERT-Check on LDAP-Server
+            #ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)
```

Comment:
I don't see how a normal user would be able to use this. They would have to build their own docker image. If you want to add this, please make it configurable using an env variable.